slapp/transforms/image_utils.py

```diff
@@ -0,0 +1,63 @@
+from typing import Tuple
+from pathlib import Path
+from PIL import Image, ImageDraw, ImageFont
+from math import ceil
+
+
+def add_scale(image_path: Path,
+              scale_position: Tuple[int, int],
+              font_file: Path,
+              ophys_experiment_resolution: float = 10/13,
+              scale_size: int = 10,
+              fill: Tuple[int, int, int] = (0, 0, 0),
+              width: int = 5,
```

Comment:
Nit-ish: `line_weight` ("width" to me makes it seem like the width the line spans, which is determined by the resolution and scale_size).

Reply:
changed to line_weight